integration/entrypoint.sh

```diff
@@ -77,6 +77,11 @@ function retry {
     fi
 }
 
+function can_erofs_ondemand_read {
+    grep CONFIG_EROFS_FS_ONDEMAND=y /usr/src/linux-headers-"$(uname -r)"/.config
```

Comment:
oh, okay, I think we can just check whether /dev/cachefiles exists

Reply:
I am afraid it's not enough since `/dev/cachefiles ` can still work for netfs on-demand read

Reply:
`CONFIG_EROFS_FS_ONDEMAND=m` is also valid.

Reply:
Fixed. Thanks